nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/js/jquery/propertytable/jquery.propertytable.js

```diff
@@ -2028,17 +2045,26 @@
                         var newPropertyDialogMarkup =
                             '<div id="new-property-dialog" class="dialog cancellable small-dialog hidden">' +
                                 '<div class="dialog-content">' +
-                                    '<div>' +
-                                    '<div class="setting-name">Property name</div>' +
+                                    '<div class="setting">' +
+                                        '<div class="setting-name">Property name</div>' +
                                         '<div class="setting-field new-property-name-container">' +
                                             '<input class="new-property-name" type="text"/>' +
                                         '</div>' +
                                     '</div>' +
+                                    '<div class="setting">' +
+                                        '<div class="setting-field new-property-sensitive-value-container">' +
+                                            '<div class="setting-name">Sensitive Value</div>' +
```

Comment:
Can we add a tooltip icon here to provide guidance regarding this feature? Specifically, when it's supported and when the user is able to drive the sensitivity of the dynamic property.

Reply:
Thanks for the suggestion, will add the icon with an explanation.